content/guides/outliers.md

```diff
@@ -32,9 +32,9 @@ For example, here is a graph of gunicorn requests by host with outlier detection
 
 You can see that one of the series is an outlier: it is handling significantly lower traffic than the others for the time window in question.
 
-During the beta period, to set up an outlier detection graph for your data you add two copies of the same metric to the graph: one showing all series in the group in a lightweight, greyscale color palette, and the other filtering to show outlier series only in a bold, warm color palette.
+To set up an outlier detection graph for your data you add a metric to the graph showing all series in the groups. You apply the outlier detection algorithm by adding "outliers" function on your data. After applying the function outliers will be colored with a bold, warm palette, while all other series will be colored with a lightweight, greyscale color pallete.
```

Comment:
Add comma and "series" for clarity: "After applying the function, outlier series"